pysheaf.py

```diff
@@ -10,19 +10,20 @@
 import matplotlib.pyplot as plt
 import networkx as nx
 import scipy.optimize
+import copy
 
 ## Data structures
 class Coface: 
     """A coface relation"""
-    def __init__(self,index,orientation):
+    def __init__(self,index=None,orientation=None):
```

Comment:
A Coface (or CofaceRelation) can only be constructed inside of an existing CellComplex, otherwise the orientation doesn't make sense. So the actual instantiations of cofaces will occur inside of the CellComplex class methods. After the relations are made, then the cofaces and faces can be added to the individual cells.